packages/editor/src/components/rich-text/index.js

```diff
@@ -79,6 +80,18 @@ export class RichText extends Component {
 			this.multilineWrapperTags = [ 'ul', 'ol' ];
 		}
 
+		if ( this.props.onSplit ) {
+			this.onSplit = this.props.onSplit;
+
+			deprecated( 'wp.editor.RichText onSplit prop', {
+				plugin: 'Gutenberg',
+				version: '4.6',
```

Comment:
I think we should warn here but not specify any version, as we won't remove this until we come up with the new deprecation policy. The API freeze is past now.

Reply:
Done in 0c1d7ce.